src/ansys/geometry/core/designer/component.py

```diff
@@ -431,6 +439,9 @@ def extrude_sketch(
         """
         # Sanity checks on inputs
         distance = distance if isinstance(distance, Distance) else Distance(distance)
+        if direction not in ("+z", "-z"):
```

Comment:
Nitpicking, but do you think it would be better to convert this variable to boolean? I think it is better for processing afterwards
```suggestion
        if direction == "+z":
            direction_bool = True
        elif direction == "-z":
             direction_bool = False
        else:
            print("Direction invalid, default +z")
            direction_bool = True
```

Reply:
I'd say that the Pythonic way to perform a string search in a list is like I had - pinging @jorgepiloto for feedback as well.

Reply:
This will change based on the enum implementation.